packages/terra-application-links/src/list/ApplicationList.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import List from 'terra-list';
+import { NavLink } from 'react-router-dom';
+import styles from './ApplicationList.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Navigational links that will generate list items that will update the path. These paths are matched with react-router to selection.
+   */
+  links: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.string,
+    path: PropTypes.string.isRequired,
+    text: PropTypes.string.isRequired,
+  })),
+};
+
+const defaultProps = {
+  links: [],
+};
+
+const ApplicationList = ({
+  links,
+  ...customProps
+}) => {
+  const listItems = links.map(link => (
+    <List.Item
+      content={
+        <NavLink aria-label={link.text} className={cx(['list-item'])} id={link.id} to={link.path} key={link.path} activeClassName={cx(['selected'])}>
+          {link.text}
+        </NavLink>
+      }
+      key={link.path}
+    />
+  ));
+
+  return (
+    <List {...customProps} className={cx(['list'])}>
+      {listItems}
+    </List>
+  );
+};
+
+ApplicationList.propTypes = propTypes;
+ApplicationList.defaultProps = defaultProps;
+
+export default ApplicationList;
```

Comment:
Why the name Application List instead of Application Menu?

Reply:
We talked about it yesterday, whether or not it should be ApplicationMenuLinks/ApplicationHeaderLInks or ApplicationList/ApplicationTabs, no one was strongly in favor, so we just took a vote and decided to go with List/Tabs.